source/fundamentals/crud/read-operations/project.txt

```diff
@@ -0,0 +1,169 @@
+==============================
+Specify Which Fields to Return
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify whether fields in a
+document should return from read operations.
+
+Sample Data
+~~~~~~~~~~~
+
+Run the following snippet to load the documents into the ``ratings``
+collection of the ``tea`` database:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/projection.go 
+   :language: go
+   :dedent:
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+.. include:: /includes/fundamentals/tea-sample-data-ending.rst
+
+Project
+-------
+
+To specify whether fields in a document should return, pass an interface
+specifying the field and ``0`` or ``1`` to the ``SetProjection()``
```

Comment:
Do you think we need an explanation that ``0`` means the field is not included and ``1`` means that is included? I see that this information is included in the sections further down "Exclude a Field" and "Include a Field", but I'm wondering if we should move it further up to here where we kind of introduce the syntax. As currently written, when I read this section, I'm wondering what the `0` and `1` correspond to.